src/spanningTree/spanningTree_driver.cpp

```
@@ -0,0 +1,264 @@
+            auto emptyresults = pgrouting::only_root_result(roots);
+            return_count = get_tuples(emptyresults, return_tuples);
+            return;
+        }
+        hint = "";
+
+        std::vector<MST_rt> results;
+        std::deque<Path> paths;
+        std::vector<std::map<int64_t, int64_t>> depths;
+
+        if (directed) {
+            switch (which) {
```

Cppcheck (2.19.0)

[style] The function 'graph_to_tuple' is never used.

(unusedFunction)